src/app/main-page/main-page.controller.js

```diff
@@ -21,8 +24,65 @@ export default class MainPageController {
             .catch(response => {
                 console.log('There are no nearby cities');
             });
-        console.log('Cities: ', this.nearbyCities);
+        this.openWeatherMapsService.getCurrentWeather(this.param)
+            .then(response => {
+                console.log(response);
+                let cont = -1;
+                let name = response.city.name;
+                let country = response.city.country;
+                response.list.forEach(element => {
+                    let flag = false;
+                    const day = new Date(element.dt * 1000).getDate();
+                    this.days.forEach(item => {
+                        if (item.day === day) {
+                            flag = true;
+                        }
+                    });
+
+                    let value = this.nameDays[new Date(element.dt * 1000).getDay()];
+
+                    if (!flag) {
+                        cont++;
+                        this.days.push({
+                            main: element.dt * 1000,
+                            day: day,
+                            dayTitle: value,
+                            city: {
+                                name: name,
+                                country: country,
+                            },
+                            hours: []
+                        })
+                    };
+
+                    const hourObject = {
+                        hour: (element.dt * 1000),
+                        temperature: {
+                            main: element.main.temp,
+                            temp_max: element.main.temp_max,
+                            temp_min: element.main.temp_min
+                        },
+                        humidity: element.main.humidity,
+                        weather: {
```

Comment:
Here, I think you can simply do: 
`weather: element.weather[0]`

Reply:
But there is also the "id", and I dont't want it